system/CLI/CLI.php

```diff
@@ -150,21 +150,16 @@ public static function init()
             // much more bash-like.
             // http://www.php.net/manual/en/readline.installation.php
             static::$readline_support = extension_loaded('readline');
-
             // clear segments & options to keep testing clean
             static::$segments = [];
             static::$options  = [];
-
             // Check our stream resource for color support
             static::$isColored = static::hasColorSupport(STDOUT);
-
             static::parseCommandLine();
-
             static::$initialized = true;
-        } else {
-            // If the command is being called from a controller
-            // we need to define STDOUT ourselves
-            define('STDOUT', 'php://output'); // @codeCoverageIgnore
+        } elseif (! defined('STDOUT')) {
+            define('STDOUT', 'php://output');
+            // @codeCoverageIgnore
```

Comment:
I changed it to 

```php
// @codeCoverageIgnoreStart
define('STDOUT', 'php://output');
// @codeCoverageIgnoreEnd
```

because php-cs-fixer is trying to add after line comment to the new line. 

Or do we not need the codeCoverageIgnore annotation at all?

Reply:
put `@codeCoverageIgnore` beside define

define(...); @codeCoverageIgnore